src/components/PeopleAndRoles/CurrentDirectors.vue

```diff
@@ -5,61 +5,62 @@
       <!-- Header -->
       <div class="header-container">
         <v-icon color="app-dk-blue">mdi-account-multiple-plus</v-icon>
-        <label class="font-weight-bold pl-2">Current Directors</label>
+        <label class="font-weight-bold pl-2">Directors</label>
       </div>
 
       <!-- Instructional Text -->
       <div class="instructional-text pt-10 px-4">
         To change directors, please use the Change feature in the Current Directors list on you business dashboard.
       </div>
 
-      <div class="list-container px-4 pt-15">
+      <v-simple-table class="director-table pt-15">
```

Comment:
OK, that's neat. I think this is our first use of a v-simple-table. I think there will be other opportunities to use one. Thanks for leading the way :)